scripts/changelog.tmpl

```diff
@@ -66,3 +66,6 @@
 * {{. }}
 {{end -}}
 {{- end -}}
+
+**Docker Image Perma Link**:
+{{ expandenv "ghcr.io/$GH_ORG/$GH_REPO:$RELEASE_SHA" }}
```

Comment:
> Can we use the actual version tag in the link? The ${RELEASE_SHA} would be very confusing (should we want anything static, it should be docker image digest, but I think the release version makes most sense).

Reply:
Is that environment variable even exposed?

Reply:
I think GITHUB_REF_NAME should do it @stepanstipl as defined here:

https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
I think so 🤞 _(I can never quite tell with GH actions until really trying it)_